src/main/java/org/team1540/robot2025/SimState.java

```diff
@@ -90,4 +96,12 @@ public double[] getSimulationOdometryTimestamps() {
         }
         return odometryTimestamps;
     }
+
+    public void addIntakeData(Rotation2d pivotPosition) {
+        if (pivotPosition.getDegrees() < 30) {
+            intakeSim.startIntake();
+        } else {
+            intakeSim.stopIntake();
+        }
+    }
```

Comment:
I'm no expert but shouldn't this be based on whether the intake is running or not, rather than whether the intake is deployed?

Reply:
RN its for collision detection, we're not actually simulating gamepiece acquisition. Activating the intake adds it to the robot hitbox